bin/smoke_test

```diff
@@ -1,4 +1,5 @@
-#!/bin/bash -euo pipefail
+#!/bin/bash
+set -euo pipefail
```

Comment:
So CircleCI's bash was very picky about this...... this will fail until this code itself is promoted to INT unfortunately :[

Reply:
I'm gonna pull this into a separate PR so it can go live first

Reply:
https://github.com/18F/identity-idp/pull/3897